README.md

```diff
@@ -1,13 +1,21 @@
-[![CircleCI](https://circleci.com/gh/apostrophecms/event/tree/master.svg?style=svg)](https://circleci.com/gh/apostrophecms/event/tree/master)
-[![Chat on Discord](https://img.shields.io/discord/517772094482677790.svg)](https://chat.apostrophecms.org)
-
 <p align="center">
-  <a href="https://github.com/apostrophecms/event">
-    <!-- TODO:  -->
-    <img src="https://raw.githubusercontent.com/apostrophecms/apostrophe/main/logo.svg" alt="ApostropheCMS logo" width="80" height="80">
-  </a>
+  <img src="https://raw.githubusercontent.com/apostrophecms/apostrophe/main/logo.svg" alt="ApostropheCMS logo" width="80" height="80">
 
   <h1 align="center">Apostrophe Events</h1>
+  <p align="center">
```

Comment:
do we need this second p wrapper?

Reply:
sort of? h1 is block, so they shouldn't go inline with that, but they're inline so should have a block level thing around them.